lib/header_internal.h

```diff
@@ -20,17 +20,17 @@ struct entryInfo_s {
 
 typedef struct hdrblob_s * hdrblob;
 struct hdrblob_s {
-    int32_t *ei;
-    int32_t il;
-    int32_t dl;
+    uint32_t *ei;
+    uint32_t il;
+    uint32_t dl;
     entryInfo pe;
-    int32_t pvlen;
+    uint32_t pvlen;
     uint8_t *dataStart;
     uint8_t *dataEnd;
 
     rpmTagVal regionTag;
-    int32_t ril;
-    int32_t rdl;
+    uint32_t ril;
+    uint32_t rdl;
```

Comment:
Unrelated, but these short variable names are quite painful to read, not just here but pretty much everywhere

Reply:
I know. These are historical, and I don't even know what some of these internal acronyms (eg pe, pvlen) exactly stand for. The idea was to keep the existing names while refactoring in progress and then at least document, but looks we're still in the refactoring phase :D